### PR TITLE
Fix `clear_untrusted_proxy_headers` deprecation warning

### LINKED ISSUE #77: deprecation warning
Running the test suite shows the following deprecation warning:

```
/home/jugmac00/Projects/plone.testing/.tox/py38/lib/python3.8/site-packages/waitress/adjustments.py:449: DeprecationWarning: In future versions of Waitress clear_untrusted_proxy_headers will be set to True by default. You may opt-out by setting this value to False, or opt-in explicitly by setting this to True.
  warnings.warn(
```

### PR DESCRIPTION
**Pull request by jugmac00**
Fixes...

DeprecationWarning: In future versions of Waitress
clear_untrusted_proxy_headers will be set to True by default. You may
opt-out by setting this value to False, or opt-in explicitly by setting
this to True.

fixes #77

Fix is analogous to https://github.com/plone/plone.recipe.zope2instance/issues/142

### CODE REVIEW

**Comment by jugmac00** [comment]
@jenkins-plone-org please run jobs

**Review by icemac — APPROVED** [review]
LGTM.

This not only prevents the deprecation warning when using this package but also when running its own tests.

**Comment by jugmac00** [comment]
@mauritsvanrees Could you have a look before I merge this?

**Review by mauritsvanrees — APPROVED** [review]
LGTM.

One of the PR jobs failed, but that was a while ago and I cannot see why this was. I restarted it.
Fine to merge if this is green.

**Comment by jugmac00** [comment]
@mauritsvanrees There is one error - but not in this package's test suite, but in the test suite of `plone.restapi`.

plone.restapi.tests.test_services_types.TestServicesTypes.test_types_document_patch_one_field

```
'Author e-mail' != None

  File "/srv/python2.7/lib/python2.7/unittest/case.py", line 329, in run
    testMethod()
  File "/home/jenkins/workspace/pull-request-5.2/src/plone.restapi/src/plone/restapi/tests/test_services_types.py", line 260, in test_types_document_patch_one_field
    self.assertEqual("Author e-mail", response.json().get("title"))
  File "/srv/python2.7/lib/python2.7/unittest/case.py", line 513, in assertEqual
    assertion_func(first, second, msg=msg)
  File "/srv/python2.7/lib/python2.7/unittest/case.py", line 506, in _baseAssertEqual
    raise self.failureException(msg)
```

I am not familiar with Jenkins and why even another package's test suite runs here.

Also I cannot judge whether the error has anything to do with the deprecation fix here.

**Review by ale-rt — APPROVED** [review]

**Comment by jugmac00** [comment]
@ale-rt  Thanks so much for your help!

**Pull request merged**
